client/fraud-scripts/forter.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
+
+export default ( { site_id } ) => {
+	// IDs in HTML can't start with a number, so calling querySelector('script#123456') breaks.
+	if ( ! document.querySelector( 'script[id="' + site_id + '"]' ) ) {
+		const script = document.createElement( 'script' );
+		// Add the ID in case the Forter script looks for it in the page.
+		script.id = site_id;
+		// TODO: This is sent to the checkout JS bundle too. That's not ideal, it's pretty big.
```

Comment:
Do you see an easy way to address this? I think for now we're only going to track admin pages, right?

Reply:
Not an easy way. After we choose one fraud provider I'll fix this system so it's less over-engineered. This was not my best idea :/